src/olympia/activity/models.py

```diff
@@ -317,7 +331,7 @@ def review_log(self):
             action__in=constants.activity.LOG_REVIEWER_REVIEW_ACTION
         ).exclude(user__id=settings.TASK_USER_ID)
 
-    def total_ratings(self, theme=False):
+    def total_reviews(self, theme=False):
```

Comment:
ahah oops!

Reply:
is that even used though ? I wonder if we could remove it (and the other one below)

Reply:
I don't think it's used.  I'll remove it.
Though that likely applies to a lot of the utility functions on `ActivityManager`,

Reply:
Could be in a separate issue/PR if you find too much stuff to remove

Reply:
https://github.com/mozilla/addons-server/pull/20028/commits/dda5e2a38fb7466113c770d8a7bbe8e7449f89a6